backend/Dockerfile.prod

```diff
@@ -0,0 +1,20 @@
+# pull official base image
+FROM python:3.7.8-alpine
+
+
+# install dependencies
+RUN pip install --upgrade pip
+COPY ./requirements.txt .
+RUN pip install -r ./requirements.txt
+
+# copy project
+COPY ./src ./src
+COPY ./nginx ./nginx
+COPY ./.env ./

```

Comment:
I don't think you need to copy `./nginx` or `./.env` into the image.

Reply:
Removed these lines.